eng/common/pipelines/templates/steps/verify-samples.yml

```diff
@@ -5,15 +5,11 @@ parameters:
   - name: ScriptDirectory
     type: string
     default: eng/common/scripts
-  - name: Condition
-    type: boolean
-    default: succeeded()
 
 steps:
   - pwsh: |
       # If the last path segment is an absolute path it will be used entirely.
-      $root = [System.IO.Path]::Combine('$(Build.SourcesDireectory)', 'sdk', '${{ parameters.ServiceDirectory }}')
-      Get-ChildItem $root -Filter *.md -Recurse | ${{ parameters.ScriptDirectory }}\Test-SampleMetadata.ps1 -AllowParentProducts
+      $root = [System.IO.Path]::Combine('$(Build.SourcesDirectory)', 'sdk', '${{ parameters.ServiceDirectory }}')
+      Get-ChildItem $root -Filter *.md -Recurse | ${{ parameters.ScriptDirectory }}/Test-SampleMetadata.ps1 -AllowParentProducts
     displayName: Verify sample metadata
     workingDirectory: $(Build.SourcesDirectory)
-    condition: ${{ parameters.Condition }}
```

Comment:
I would have expected this condition approach to work do you know what broke?

Reply:
What broke is that the parameter was declared as a `boolean` type. I thought I effectively copied this from another template, but that template simply used:

```yaml
parameters:
  # ...
  Condition: succeeded()
```

In hindsight, this probably defaults to a `string` type; but, in this case, given how frustratingly slow the pipelines were running last night (it seemed), I figured it wasn't needed right now so I just removed it. We can always test with a `string` type (and assume it will cast to a `boolean` when used in this line you've highlighted) later if needed.